style.css

```diff
@@ -167,60 +202,24 @@ ul li:hover ul {
 
 /* style for slider ends here */
 
-
-.box1 {
-  box-sizing: border-box;
-  grid-column-start: 1;
-  grid-column-end: span 2;
-  padding-bottom: 10px;
-  height: 700px;
-  width: 2000px;
-  background-color: #3750A5;
-  color: #A2091B;
-}
-
-.box2 {
-  box-sizing: border-box;
-  grid-column-start: 1;
-  grid-column-end: span 2;
-  padding-bottom: 10px;
-  height: 20px;
-  width: 2000px;
-  background-color: #4DC9FF;
-  color: #A2091B;
-}
-
-.box3 {
-  box-sizing: border-box;
-  grid-column-start: 1;
-  grid-column-end: span 2;
-  padding-bottom: 10px;
-  height: 20px;
-  width: 2000px;
-  background-color: #87CEEB;
-  color: #FBCA01; 
-}
-
-.sidebar {
-	background-color: #6FC0DB;
-	height: 1500px;
-	width: 200px;
-	float: left;
-}
-
 .content{
 	box-sizing: border-box;
-	background-color: #FFFFFF;
+	background-color: #87CEEB;
 	position: relative;
-	width: 2000px;
-	height: 1500px;
-	left: 200px;
-	color: #000000;
+	width: 1990px;
+	height: 1300px;
+	font-size: 12px;
+	text-align: left;

```

Comment:
In my first version I added an aside bar I and moved the content 200px to the left. I got feedback that I should get rid of the asidebar its not holding any images or placeholder text. So I deleted the aside and got rid of the left: 200px and made the width 1990px. Also for the height I thought 1500px was to long I changed it to 1300px so students/caregivers don't have to scroll down a lot. For the placeholder text I moved the text to the left side of the website.